pkg/sources/github/receive_adapter/receive_adapter.go

```diff
@@ -54,8 +57,26 @@ type GithubSecrets struct {
 // HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
 func (h *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
 	log.Print("Handling Pull Request")
-	pl := payload.(github.PullRequestPayload)
-	postMessage(h.target, &pl)
+
+	hdr := http.Header(header)
```

Comment:
I think a webhooks.Header is an http.Header, so it should already have `.Get()`.

Reply:
I do not have access to .Get() from the webhooks.Header object for some reason, it was a compile error when I was trying.

Reply:
Aha -- http.Header _is_ an `http.Header`, which is a `map[string[]string`. But since it has a different type name, methods which take an `http.Header` as a receiver don't apply to `webhooks.Header`.

So glad they "minimize imports": https://github.com/go-playground/webhooks/blob/v3.13.0/webhooks.go#L8

It looks like this declaration was removed in v5.